pom.xml

```diff
@@ -218,37 +217,31 @@
      -->
     <profile>
       <id>snapshots-and-staging</id>
-      <properties>
-        <!-- override point for ASF staging/snapshot repos -->
-        <asf.staging>https://repository.apache.org/content/groups/staging/</asf.staging>
-        <asf.snapshots>https://repository.apache.org/content/repositories/snapshots/</asf.snapshots>
-      </properties>
-
+      <!-- override point for ASF staging/snapshot repos -->
       <pluginRepositories>
         <pluginRepository>
           <id>ASF Staging</id>
-          <url>${asf.staging}</url>
+          <url>https://repository.apache.org/content/groups/staging/</url>
```

Comment:
This parameterization was added in #569 and still might be useful. Does it need to be removed?

Reply:
Yes, Maven 4.0 enforces constants in repo URIs.

Reply:
Understood. FYI for @steveloughran who originally put in #569.